db/schema.go

```diff
@@ -103,8 +104,29 @@ type TenantSPEligibilityClauses struct {
 	TenantID        ID                 `json:"tenant_id" gorm:"primaryKey"`
 	ClauseAttribute string             `json:"attribute" gorm:"primaryKey"`
 	ClauseOperator  ComparisonOperator `json:"operator" gorm:"type:comparison_operator;not null"`
-	ClauseValue     string             `json:"value" gorm:"not null"`
+	ClauseValue     StringSlice        `json:"value" gorm:"type:JSON;not null"`
 }
+
+type StringSlice []string
```

Comment:
can you add a comment just explaining why we had to do this (no support in Gorm for array types)